code/game/gamemodes/game_mode.dm

```diff
@@ -507,6 +503,16 @@
 		if(P.client && P.ready == PLAYER_READY_TO_PLAY)
 			. ++
 
+////////////////////////////
+//Keeps track of all living//
+////////////////////////////
+
+/datum/game_mode/proc/get_living_crew()

```

Comment:
Why is this under /datum/game_mode instead of just being a global proc? It doesn't use anything from gamemode. Just make a helper proc.

Reply:
Yea sure thing, I'll replace the other pieces of code that use the code there too

Reply:
I put it in the mobs.dm helper file, right place?